public/index.css

```diff
@@ -260,6 +260,23 @@
   #header {
     text-align: center;
   }
+  .headerH1 {
+    font-size: 3.3rem;
+  }
+
+  .headerH3 {
```

Comment:
I suggest we move any changes to the headers to instead modify the default h1 and h3 values defined at the top of the file. I'm not convinced we need to changed what they were though.